src/lib/components/OpenWebSearchResults.svelte

```diff
@@ -1,15 +1,16 @@
 <script lang="ts">
 	import type { WebSearchUpdate } from "$lib/types/MessageUpdate";
-	import CarbonError from "~icons/carbon/error-filled";
 
+	import CarbonError from "~icons/carbon/error-filled";
 	import EosIconsLoading from "~icons/eos-icons/loading";
+	import IconInternet from "./icons/IconInternet.svelte";
```

Comment:
on the above lines, you can see that we use the icons directly from libs when possible
```html
import CarbonError from "~icons/carbon/error-filled";
import EosIconsLoading from "~icons/eos-icons/loading";
```

Similarly, would it be possible not to manually define `IconInternet`. Instead use it like from "~icons/xyz/internet" ?

Reply:
It's a custom made icon (not from iconify)